torch/onnx/utils.py

```diff
@@ -483,6 +483,26 @@ def _export_to_pretty_string(model, args, f, export_params=True, verbose=False,
                                         operator_export_type, google_printer,
                                         val_keep_init_as_ip, custom_opsets, val_add_node_names)
 
+def _diagnose_export(model, args, f, verbose=False, training=TrainingMode.EVAL,
```

Comment:
Please add comments explaining the api, similarly to how `export` is commented.

Reply:
Do I need to add comments in the __init__ file as well?

Reply:
Yes, let's put the comments in init file.